samples/net/sockets/big_http_download/sample.yaml

```diff
@@ -1,10 +1,16 @@
 sample:
   description: BSD Sockets big HTTP download example
   name: big_http_download
+common:
+  filter: TOOLCHAIN_HAS_NEWLIB == 1
+  harness: net
+  min_ram: 32
+  min_flash: 128
+  tags: net socket
 tests:
   sample.net.sockets.big_http_download:
-    filter: TOOLCHAIN_HAS_NEWLIB == 1
     harness: net
```

Comment:
The individual test section cannot be left empty so I was just wondering if we could set here for example some config option so that the harness setting could be removed.

Reply:
> The individual test section cannot be left empty

Yes. I looked into that, pykwalify (following limitation of upstream kwalify scheme doesn't allow that, e.g. https://github.com/Grokzen/pykwalify/issues/149).

The normal YAML syntax then would be: `sample.net.sockets.big_http_download: {}`, but I considered that too magic for not YAML lawyers among us, and used
~~~
sample.net.sockets.big_http_download:
   harness: net
~~~

as a placeholder.

Reply:
What about adding
```
    extra_configs:
      - CONFIG_POSIX_API=n
```
which is a "dummy" setting in this case, instead of the `harness:net`?

Reply:
> which is a "dummy" setting in this case, instead of the harness:net

I can do that, but are you sure it will be improvement? (I'd use `CONFIG_NET_SOCKETS_POSIX_NAMES=y` then however, to show what we test with, not what without.)